package.json

```diff
@@ -77,6 +77,9 @@
     "through2": "^3.0.1",
     "yargs": "^17.6.2"
   },
+  "engines": {
+    "node": ">=14.0.0 <18"
```

Comment:
Should we:

```suggestion
    "node": ">=16.0.0 <18"
```

Reply:
I tend to opt for permissive for flexibility. Our project does work with 14.